envs/environment.yaml

```diff
@@ -8,12 +8,12 @@ channels:
   - bioconda
   - http://conda.anaconda.org/gurobi
 dependencies:
-  - python
+  - python>=3.8
```

Comment:
Is this necessary? `python=3.7` has its EOL in 2023. If it is necessary this also has to be in the release notes.

Reply:
is done